service/controller/v25/cloudformation/spec.go

```diff
@@ -2,9 +2,9 @@ package cloudformation
 
 import "github.com/aws/aws-sdk-go/service/cloudformation"
 
-// CloudFormationInterface provides a set of methods to work with
```

Comment:
The file got renamed. I am fine having `types.go`, but then it should contain types. This here is interface and so far we have it in `spec.go`.

Reply:
+1